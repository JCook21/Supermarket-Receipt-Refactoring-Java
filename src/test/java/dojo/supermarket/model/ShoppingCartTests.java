package dojo.supermarket.model;

import org.approvaltests.Approvals;
import org.junit.Before;
import org.junit.Test;

public class ShoppingCartTests
{
	private ShoppingCart shoppingCart;

	@Before
	public void setup()
	{
		shoppingCart = new ShoppingCart();
	}

	@Test
	public void addOneNewProduct()
	{
		Product newProduct = new Product("Lucky Charms", ProductUnit.Each);

		shoppingCart.addItemQuantity(newProduct, 1.0);

		Approvals.verify(shoppingCart.productQuantities());
	}
}
