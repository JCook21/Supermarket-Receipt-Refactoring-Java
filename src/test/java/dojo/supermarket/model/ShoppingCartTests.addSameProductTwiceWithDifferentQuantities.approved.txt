Product{name='Cheerios ', unit=Each} : 3.0 
